### PR TITLE
updated general settings docs

### PR DESCRIPTION
**Pull request by Robbt**
This updates the general settings documentation to include all options currently provided by LibreTime in the order that they are displayed on the screen.

### CODE REVIEW

**Comment by Robbt** [comment]
I want to update the screenshot before we merge this.

**Comment by Robbt** [comment]
new screen shots have been added so this should be ready to merge.

File: docs/manual/preferences/index.md [hunk 1]
```diff
@@ -1,11 +1,39 @@
...
+
+ ![](static/api-settings.png)
+
+You can enable live, read-only access to the Libretime schedule calendar for your station's public website with the **Public LibreTime API** option, if you wish. (There is more about this feature in the *Exporting the schedule* chapter, in the *Advanced Configuration* section of this book).
+
+The **Allowed CORS URLs** is intended to deal with situations where you want a remote site with a different domain to access the API.
+
+ The **Display login button on your Radio Page?** will determine whether visitors to your site see a link to login. If this is disabled DJs and admins will need to goto http://SITEURL/login to be able to login. 
+ 
+ The **Tune-In Settings** section is intended for stations that have partnered with TuneIn to automatically push their now playing metadata to TuneIn. This hasn't been tested and also requires special credentials from parterning with TuneIn to test.
```
Comment (paddatrapper) [inline docs/manual/preferences/index.md]: s/parterning/partnering

Comment (paddatrapper) [reply docs/manual/preferences/index.md]: Fixed

**Comment by Robbt** [comment]
Thanks for this.

**Pull request merged**